lib/tasks/assets.rake

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-EMBER_CLI = ENV["EMBER_CLI_PROD_ASSETS"] != "0"
+EMBER_CLI ||= ENV["EMBER_CLI_PROD_ASSETS"] != "0"
```

Comment:
This will still cause it to be redefined if the result is `false`, right?

```suggestion
EMBER_CLI = ENV["EMBER_CLI_PROD_ASSETS"] != "0" if EMBER_CLI.nil?
```

Reply:
Yes you are right indeed :+1: